doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -65,13 +72,17 @@ To provision the certificates and the private key to the nRF9160 modem, complete
 #. Build and program the :ref:`at_client_sample` sample to the nRF9160-based device as explained in :ref:`gs_programming`.
 #. Launch the `LTE Link Monitor`_ application, which is part of `nRF Connect for Desktop`_.
 #. Click :guilabel:`Certificate manager` located at the top right corner.
-#. Copy the Baltimore CyberTrust Root certificate into the ``CA certificate`` entry.
+#. Copy the server root certificate into the ``CA certificate`` entry.
 #. Copy and paste the device certificate and the key created using the scripts located in `Creating Azure IoT Hub certificates`_, into the respective entries (``Client certificate``, ``Private key``).
-#. Select a desired security tag (any positive integer, for example, ``42``) and click :guilabel:`Update certificates`.
+#. Select a desired security tag (any positive integer in the range 0 to 2147483647) and click :guilabel:`Update certificates`.
 
 .. note::
    The chosen security tag while provisioning the certificates must be same as the security tag configured by the :kconfig:option:`CONFIG_AZURE_IOT_HUB_SEC_TAG` option.
 
+.. note::
+   If more than one root server certificate is used, the second one can be provisioned to a different security tag and configured in the application using :kconfig:option:`CONFIG_AZURE_IOT_HUB_SECONDARY_SEC_TAG`.
```

Comment:
Should the second root certificate also contain the same «Client certificate» and «Private key»? Or can those be empty for the second root CA?

Reply:
Root certificates never contain client certificates or private keys.

Reply:
I meant in the UI. It looks like they are stored in the same security tag:
![Screenshot 2022-08-10 at 10 11 59](https://user-images.githubusercontent.com/149725/183850336-fa1e2ff3-5974-4a68-a8a8-324fe56da881.png)

When thinking more about it, is the server root certificate only used to validate the server's authenticity? It doesn't make sense that the client certificates generated for test purposes is signed by a public root CA. From the UI it looks like it has something to do with the client certificates.

Reply:
Yes, they go in the same secTag. The root certificate is part of the `CA certificate`, which can contain multiple certificates (root + CA root + intermediate CA), and yes it's needed to verify the endpoint authenticity.

Reply:
I've asked internally if it is sufficient enough to only store a new CA into the second sec tag. Both the primary and secondary sec tags are passed into the modem in a sec tag list. So I wouldn't be surprised if all certificates are handled in the same context on the modem.

Reply:
It is sufficient to provision the DigiCert certificate to the secondary slot to be able to connect with an invalid CA in the primary. This is also mentioned in line 86.